Makefile.toml

```diff
@@ -129,6 +129,26 @@ find . -not -path '*/node_modules/*' -name '*.js' | xargs ./api_tests/node_modul
 '''
 ]
 
+[tasks.cargo-toml-format]
+description = "Runs cargo tomlfmt to format Cargo.toml files."
+workspace = false
+install_crate = { crate_name = "cargo-tomlfmt", binary = "cargo-tomlfmt", test_arg = "--help" }
```

Comment:
this may not be needed

Reply:
It's just installing `cargo-tomlfmt` if it can't find it, right? Do we not want this to happen?

Reply:
I think the attribute has defaults but I am not sure they apply here.

Reply:
as explained IRL. cargo-make may be clever enough not to need this line